plsync/mlabconfig_test.py

```diff
@@ -25,6 +84,8 @@ def setUp(self):
                                      self.users,
                                      nodegroup='MeasurementLabCentos')]
         self.attrs = [model.Attr('MeasurementLabCentos', disk_max='60000000')]
+        # Turn off logging output during testing (unless critical).
+        logging.disable(logging.CRITICAL)
```

Comment:
Is the argument to disable() inclusive? Should this be `logging.disable(logging.ERROR)` if we still want to output CRITICAL messages?